### DISCUSSION TITLE
Foundry Compiling Issue

### DISCUSSION THREAD

**Discussion by Rayyan-Uddin** [Cyfrin/foundry-full-course-cu#2412] [Q&A] [answered]
Hi, Im getting this message in terminal when running "forge build or forge compile" command.
Help please!

![errr](https://github.com/user-attachments/assets/0551b9d6-96de-47f7-94a5-6714f258a96d)

**Comment by EngrPips** [comment, marked as answer]
run `cd foundry-simple-storage` then run `forge compile` lets see if that works

**Reply by Rayyan-Uddin** [reply]
Yup, fixed.
Thanks mate.

**Reply by EngrPips** [reply]
You are welcome, Keep crushing it.
